src/riddle/useCases/solveRiddle/__tests__/createSolveRiddleService.test.ts

```diff
@@ -0,0 +1,126 @@
+
+import { describe, it, expect } from 'vitest';
+import { createSolveRiddleModel } from '../createSolveRiddleService';
+
+const mockRiddle = {
+  id: 'riddle-1',
+  contents: 'What has keys but can’t open locks?',
+  answers: [
+    { id: 'a', text: 'Piano' },
+    { id: 'b', text: 'Map' },
+    { id: 'c', text: 'Locksmith' },
+  ],
+};
+
+describe('createSolveRiddleModel', () => {
+  it('returns true for shouldDisplayRiddle ', () => {
+    const model = createSolveRiddleModel({
+      isLoading: false,
+      selected: undefined,
+      correct: undefined,
+      randomRiddle: mockRiddle,
+      answersAvailable: true,
+    });
+    expect(model.shouldDisplayRiddle).toBe(true);
+  });
+
+  it('returns false for shouldDisplayRiddle when isLoading is true', () => {
```

Comment:
```suggestion
  it("don't display riddle when answers are still loading', () => {
```

We formulate using English instead of referencing implementation details:
- ~`returns false`~ => `don't display riddle`
- ~`isLoading is true`~ => `answers are still loading`

(ditto throughout)